### DISCUSSION TITLE
Compose Key

### DISCUSSION THREAD

**Discussion by dumanshr** [labwc/labwc#1242] [Ideas]
I saw keyboard layout is implemented on environment file.  But how can I use compose key feature without changing my keyboard laybout? I did not see Compose Key implemented yet. I would like to use CAPS LOCK as my compose key.  Please add this feature as soon as possible.

**Comment by johanmalm** [comment]
`XKB_DEFAULT_OPTIONS=compose:caps`

grep "compose:" /usr/share/X11/xkb/rules/base.lst

https://wiki.archlinux.org/title/Xorg/Keyboard_configuration
Section 3.5.1
